### DISCUSSION TITLE
Ability to create playlist and queue based on directory.

### DISCUSSION THREAD

**Discussion by PatrykChwastek** [music-assistant/support#990] [Feature requests and ideas]
Hi, i really find useful ability to select folder in local files. And create a playlist or queue base on files inside. It is simpler and faster than searching for individual files.

**Comment by OzGav** [comment]
Please see existing thread https://github.com/music-assistant/hass-music-assistant/discussions/968

**Reply by PatrykChwastek** [reply]
Oh, yes I didn't notice it. So so as i see the idea is dropped anyway. Sorry for interrupting.

**Reply by OzGav** [reply]
Actually the last I said there was “see what others think” so you should post your support there and I will shortly remove this discussion
